tests/test_http_request.py

```diff
@@ -1217,25 +1217,17 @@ def test_from_response_css(self):
                           response, formcss="input[name='abc']")
 
     def test_from_response_valid_form_methods(self):
+        form_methods = [[method, method] for method in self.request_class.valid_form_methods] + \
+                       [['UNKNOWN', 'GET']]
```

Comment:
Note that we tend to avoid using `\` for line continuation, and instead use parentheses. For example:

```suggestion
        form_methods = (
            [[method, method] for method in self.request_class.valid_form_methods]
            + [['UNKNOWN', 'GET']]
        )
```

That said, we will eventually switch to Black I think, so it probably does not matter much.

Reply:
fixed https://github.com/scrapy/scrapy/pull/5246/commits/d3be17b87f29d91f402884bf4bf96da8af93d225